src/Asset.js

```diff
@@ -214,7 +213,7 @@ class Asset {
   }
 
   generateHash() {
```

Comment:
Is this method needed anymore?

Reply:
RawAsset uses it as it has no generated content

Reply:
Maybe we should keep the method then and just call it after post-processing is done?